tox.ini

```
@@ -84,3 +82,9 @@ commands =
            --log-cli-level=INFO \
            {posargs} \
            {[vars]tests_path}/integration
+
+[testenv:venv]
+deps =
+    -r {tox_root}/dev-requirements.txt
+    -r {tox_root}/requirements.txt
+commands = {posargs}
```

[nit]: Missing newline at the end of the file:

```suggestion
commands = {posargs}

```